tests/integration/it_utils.py

```diff
@@ -331,6 +332,16 @@ def create_amm_pool(
         issuer_wallet,
     )
 
+    # The below flag is required for AMMClawback tests
+    if issuer_account_enable_trustline_clawback:
+        sign_and_reliable_submission(
+            AccountSet(
+                account=issuer_wallet.classic_address,
+                set_flag=AccountSetAsfFlag.ASF_ALLOW_TRUSTLINE_CLAWBACK,
+            ),
+            issuer_wallet,
+        )
+
```

Comment:
Rather than doing a second AccountSet transaction, we still do one AccountSet and OR `ASF_ALLOW_TRUSTLINE_CLAWBACK` flag with `ASF_DEFAULT_RIPPLE` when we want to enable clawback.

Reply:
That doesn't work. Here is the error:
```
File "/Users/ckeshavabs/xrpl-py/xrpl/models/base_model.py", line 228, in _from_dict_single_param
    raise XRPLModelException(error_message)
xrpl.models.exceptions.XRPLModelException: set_flag expected a typing.Optional[xrpl.models.transactions.account_set.AccountSetAsfFlag], received a <class 'int'>
```

The `AccountSet.set_flag` field accepts the enum `AccountSetAsfFlag`, not an `int`

Reply:
Why not just use `AccountSet.flags` instead?

Reply:
https://xrpl.org/docs/references/protocol/transactions/types/accountset#accountset-flags

Do you know if `asf` flags can be passed into the `tf` flags input field? Based on the docs, they have a separate `set_flag` and `clear_flag` field for this purpose

Reply:
Oh that's inconvenient. Why can't they be included all in one transaction rather than separate ones? I'm not sure how rippled implemented this.

Reply:
rippled ran out of bitwise flags, lol

Reply:
I share your frustrations @khancode 
@mvadari  the flags field is a `std::uint32_t` right? We should be able to fit in `2^32 -1` values right? Am I missing something?

Reply:
I'm referring to `sfFlags`

Reply:
They're flags, so they have to be distinct bitwise. So you've only got 32, and some of that space has to be reserved for global transaction flags.

Reply:
https://xrpl.org/docs/references/protocol/transactions/common-fields#flags-field

Reply:
Ah that makes sense.. Thanks for clarifying this 😄